docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md

```diff
@@ -1,14 +1,15 @@
 We are going to create the following logical configuration. We create a
-PingSource, feeding events into the Broker, then we create a `Filter` that
-wires those events into a [`Sequence`](../../../flows/sequence.md) consisting of 3
+PingSource, feeding events into the Broker, then we create a `Filter` that wires
+those events into a [`Sequence`](../../../flows/sequence.md) consisting of 3
 steps. Then we take the end of the Sequence and feed newly minted events back
 into the Broker and create another Trigger which will then display those events.
 
 ## Prerequisites
 
-For this example, we'll assume you have set up an `InMemoryChannel` as well as Knative Serving (for our functions). The examples
-use `default` namespace, again, if your broker lives in another Namespace, you
-will need to modify the examples to reflect this.
+For this example, we'll assume you have set up an `InMemoryChannel` as well as
+Knative Serving (for our functions). The examples use `default` namespace,
```

Comment:
```suggestion
Knative Serving (for our functions). The examples use the `default` namespace,
```
namespace is used twice in this sentence, but it's capitalized in one spot and lowercase in another, not sure which one should be used.

Reply:
lowercase please